RELEASING.md

```diff
@@ -1,3 +1,69 @@
+
+#### Prepare BWC data and update BWC versions
+
+Backwards Compatibility Tests (BWC) are cypress tests that verify any changes are backwards compatible with previous versions. RM will generate test data and test locally following instructions [here](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/TESTING.md#backwards-compatibility-tests) and cut PR to include both generated data and version upgrade for automated build. (See example [PR](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2393/files))
+
+#### Cut release branch for major / minor release
+
+For major / minor release, RM will need to cut the release branch from the parent branch, [following OpenSearch project branching](https://github.com/opensearch-project/.github/blob/main/RELEASING.md#opensearch-branching)
+
+### Release Phase 2 - Pre-Release
+
+The release process for OpenSearch is centralized. Jenkins workflows are in place to run daily snapshot builds for OpenSearch and OpenSearch Dashboards. RM will update the release branch version in the distribution manifest (see example [PR](https://github.com/opensearch-project/opensearch-build/pull/2586/files)) and increment the parent branch version (see example [PR](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2295/files)).
+
+#### Write release notes
+
+OpenSearch Dashboards maintains a [CHANGELOG.md](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/CHANGELOG.md) and verifies it as part of the PR checklist step. For the time being, RM will create release notes PR with the label `doc`, referring to the `CHANGELOG.md` (see example [PR](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2318))
+
+### Release Phase 3 - Release testing
+
+#### Verify integration and BWC test results
+
+The automated integration test and BWC test are executed concurrently with the release artifacts build. RM should examine the test results and assist in triaging the broken test case.
+
+Example build:
+
+x64: https://build.ci.opensearch.org/job/integ-test-opensearch-dashboards/995/
+
+arm64: https://build.ci.opensearch.org/job/integ-test-opensearch-dashboards/996/
+
+Example test results:
+https://opensearch-project.github.io/opensearch-dashboards-functional-test/site/?version=2.3.0&build_number=4104&test_number=996
+
+Note: change `arch` to correspond to the build's processor (CPU) architecture
+
+#### Sanity test with tarball and docker image
+
+Once the release candidate artifacts are built, RM will configure the OpenSearch cluster with OpenSearch Dashboards according to the [instructions in the OpenSearch build repo](https://github.com/opensearch-project/opensearch-build/issues/2447#issuecomment-1241406594) and produce sanity tests to identify broken functionalities caused by new features / code changes. If you find any, please file bug reports and assist in triaging the bugfix.
+
+### Release Phase 4 - Release Announcement
```

Comment:
Not sure if we include that after the release announced the build repo team will trigger a job that will cut a tag in each repo from the commit hash and branch that was included in the release (which could take a couple hours for it to be available). But it is on the maintainers of the repo to create the release form the tag automatically created.

Reply:
Yes i didn't put the tag generate part since it's not owned by OSD, but I can add it here.

The second part is included in Post-Release session.